.gitignore

```diff
@@ -6,7 +6,8 @@ build/
 *.env
 config*.yaml
 config*.yml
-docs/*.md
+#TODO: verify intent of ignoring docs/*.md at PR
+#docs/*.md
```

Comment:
Is removing `docs/*.md` from the `.gitignore`, OK?
I could quite understand why we are doing it...

I'm allowing it in this PR to add a placeholder page to prevent the http 404 on this link:
https://platform-cli.docs.flanksource.com/

Reply:
It should just be to ignore the generated docs - primarily reference/config.md